source/core/owt_base/SVTHEVCEncoder.h

```diff
@@ -16,7 +16,7 @@
 #include "logger.h"
 #include "MediaFramePipeline.h"
 
-#include "EbApi.h"
+#include "svt-hevc/EbApi.h"
```

Comment:
why change this path?

Reply:
qiujiao need use svt-hevc installed in /usr,
this is the path generated after "make install" (PREFIX/include/svt-hevc/EbApi.h)